enterprise/admin/views.py

```diff
@@ -328,7 +362,16 @@ def post(self, request, customer_uuid):
             if course_details:
                 course_mode = manage_learners_form.cleaned_data[ManageLearnersForm.Fields.COURSE_MODE]
                 course_id = course_details['course_id']
-                self._enroll_users(enterprise_customer, linked_learners, course_id, course_mode, request)
+                notification_type = manage_learners_form.cleaned_data.get(ManageLearnersForm.Fields.NOTIFY)
+                notify = notification_type == ManageLearnersForm.NotificationTypes.BY_EMAIL
```

Comment:
I'd prefer to keep things open-ended and stick with the enum pattern -- I'm sure at some point someone will want Instagram support...